content/en/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -436,12 +436,13 @@ configuration](https://github.com/kubernetes/kubernetes/blob/release-1.10/pkg/sc
 **Example:**
 
 The following configuration for a scheduler policy indicates that the
-cluster-level extended resource "example.com/foo" is handled by scheduler
+cluster-level extended resource "example.com/foo" is handled by the scheduler
 extender.
- - The scheduler sends a pod to the scheduler extender only if the pod requests
-   "example.com/foo".
- - The `ignoredByScheduler` field specifies that the scheduler does not check
-   the "example.com/foo" resource in its `PodFitsResources` predicate.
+
+- The scheduler sends a Pod to the scheduler extender only if the Pod requests
```

Comment:
Nit: You don't need the hanging indent on the second line of a list item. Best practice is to line up the second line with where the text starts on the first line. I won't hold this PR for that, though.